src/common.speech/SpeechConnectionFactory.ts

```diff
@@ -5,13 +5,32 @@ import {
     ProxyInfo,
     WebsocketConnection,
 } from "../common.browser/Exports";
-import { OutputFormatPropertyName } from "../common.speech/Exports";
-import { IConnection, IStringDictionary } from "../common/Exports";
-import { OutputFormat, PropertyId } from "../sdk/Exports";
-import { AuthInfo, IConnectionFactory, RecognitionMode, RecognizerConfig, WebsocketMessageFormatter } from "./Exports";
-import { QueryParameterNames } from "./QueryParameterNames";
+import {
+    ForceDictationPropertyName,
+    OutputFormatPropertyName,
+} from "../common.speech/Exports";
+import {
+    IConnection,
+    IStringDictionary
+} from "../common/Exports";
+import {
+    OutputFormat,
+    PropertyId
+} from "../sdk/Exports";
+import {
+    ConnectionFactoryBase
+} from "./ConnectionFactoryBase";
+import {
+    AuthInfo,
+    RecognitionMode,
+    RecognizerConfig,
+    WebsocketMessageFormatter
+} from "./Exports";
+import {
+    QueryParameterNames
+} from "./QueryParameterNames";
 
-export class SpeechConnectionFactory implements IConnectionFactory {
+export class SpeechConnectionFactory extends ConnectionFactoryBase {
```

Comment:
do you want to add the newly added SendMessageSync and SetMessageProperty in the native class of connection to here as well?